src/runner/index.js

```diff
@@ -183,6 +184,16 @@ export default class Runner extends EventEmitter {
         assets.forEach(asset => this.proxy.GET(asset.path, asset.info));
     }
 
+    _validateDebugLogger () {
+        const debugLogger = this.configuration.getOption(OPTION_NAMES.debugLogger);
+
+        if (debugLogger === void 0) {
+            this.configuration.mergeOptions({
```

Comment:
strictly speaking we need to check that a custom `debugLogger` has `showBreakpoint` and `hideBreakpoint` methods